AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestExecutionStore.kt

```diff
@@ -0,0 +1,59 @@
+package dev.androidx.ci.testRunner
+
+import dev.androidx.ci.datastore.DatastoreApi
+import dev.androidx.ci.firebase.FirebaseTestLabApi
+import dev.androidx.ci.firebase.ToolsResultApi
+import dev.androidx.ci.gcloud.GoogleCloudApi
+
+internal class TestExecutionStore(
+    private val toolsResultApi: ToolsResultApi,
+    private val googleCloudApi: GoogleCloudApi,
+    firebaseProjectId: String,
+    datastoreApi: DatastoreApi,
+    firebaseTestLabApi: FirebaseTestLabApi,
+    gcsResultPath: String
+) {
+    private val testMatrixStore = TestMatrixStore(
```

Comment:
why don't we inject these in the constructor.
`TestRunnerServiceImpl` already have them so it should pass them in while creating the execution store.